test/screenshotter/ss_data.js

```diff
@@ -0,0 +1,14 @@
+var fs = require("fs");
+var querystring = require("querystring");
+var jsyaml = require("js-yaml");
```

Comment:
I tend to sort built-in modules before explicitely installed ones, but that's personal preference and I'll gladly adapt this.